akka-remote/src/main/resources/reference.conf

```diff
@@ -171,15 +167,40 @@ akka {
       log-sent-messages = off
 
       advanced {
+      
+        # Maximum serialized message size, including header data.  
+        maximum-frame-size = 256 KiB
+        
+        # Direct byte buffers are reused in a pool with this maximum size.
+        # Each buffer has the size of 'maximum-frame-size'.
+        buffer-pool-size = 128
```

Comment:
mention that it also works with a lower pool-size than used buffers, however, in that case there is a higher churn of DirectByteBuffers